src/main/java/Duke.java

```diff
@@ -1,10 +1,234 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static boolean isCompleted = false;
+    private static final List<Task> userList = new ArrayList<>();
+
+    public static void welcomeMessage() {
+        separator();
+        printMessage("Hello! I'm Duke");
+        printMessage("What can I do for you?");
+        separator();
+    }
+
+    public static void endingMessage() {
+        separator();
+        printMessage("Bye. Hope to see you again soon!");
+        separator();
+    }
+
+    public static void separator() {
+        String separatorLine = "-".repeat(60);
+        System.out.println(separatorLine);
+    }
+
+    public static void printMessage(String message) {
+        String outputMessage = String.format("| %-57s|", message);
+        System.out.println(outputMessage);
+    }
+
+    public static void checkInput(String inputMessage) {
+        String[] message = inputMessage.split(" ");
+        if (message.length == 1 && message[0].equalsIgnoreCase("bye")) {
+            isCompleted = true;
+            return;
```

Comment:
Is it better to keep checking for message.length() > 1, or in this case it is better to check for "bye" or "list" at the beginning? I am unsure also, feel free to discuss:)